packages/react-badge/src/components/PresenceBadge/usePresenceBadgeStyles.ts

```diff
@@ -3,9 +3,13 @@ import { PresenceBadgeState } from './PresenceBadge.types';
 import { useBadgeStyles } from '../../Badge';
 
 const useStyles = makeStyles({
-  root: {
+  root: theme => ({
     padding: 0,
-  },
```

Comment:
Can you show before/after screenshots? I'm curious if we need to update presence styles in other codebases...